doc/releases/squid.rst

```diff
@@ -14,11 +500,11 @@ We recommend that all users update to this release.
 Notable Changes
 ---------------
 
-* This hotfix release resolves an RGW data loss bug when CopyObject is used to copy an object onto itself. 
-  S3 clients typically do this when they want to change the metadata of an existing object. 
-  Due to a regression caused by an earlier fix for https://tracker.ceph.com/issues/66286, 
-  any tail objects associated with such objects are erroneously marked for garbage collection. 
-  RGW deployments on Squid are encouraged to upgrade as soon as possible to minimize the damage. 
+* This hotfix release resolves an RGW data loss bug when CopyObject is used to copy an object onto itself.
```

Comment:
s/hotfix//

Reply:
This is the release note for 19.2.2 ;)  @yuriw It would be better to not have the unrelated whitespace changes in the PR.